plugins/autodoc/autodoc-plugin/src/main/java/org/eclipse/edc/plugins/autodoc/AutodocDependencyInjector.java

```diff
@@ -31,21 +31,20 @@ class AutodocDependencyInjector implements DependencyResolutionListener {
     private static final String VERSION = "edc.version"; // must be identical to EdcModuleProcessor.VERSION
     private static final String ID = "edc.id"; // must be identical to EdcModuleProcessor.ID
     private static final String OUTPUTDIR = "edc.outputDir"; // must be identical to EdcModuleProcessor.EDC_OUTPUTDIR_OVERRIDE
+    private static final String DEPENDENCY_NAME = format("%s:%s", "org.eclipse.edc", "autodoc-processor");
```

Comment:
micro-nit but strictly speaking that's not a dependency, just the "GA" part in GAV. so maybe the slightly more appropriate name would be `AUTODOC_GROUP_ARTEFACT`.